### PR TITLE
Reddit integration

### PR DESCRIPTION
**Pull request by jasonrsadler**
## Changes

Adds Reddit tipping panel integration

##### Link / storybook path to visual changes
- 

https://brave-ui-gegolvmun.now.sh
## Integration
- [ ] Does this contain changes to src/components or src/
  - [ ] Will you publish to npm immediately after this PR, or wait until sometime in the future?
  - [ ] Incompatible API change to something existing _(major version increase)_
  - [ ] Adding new backwards-compatible functionality? _(minor version increase)_
  - [ ] Fixing a bug backwards-compatibly? _(patch version increase)_
  
- [ ] Does this contain changes to src/features for brave-core?
  - [ ] Are there non backwards-compatible changes required for brave-core? **Do not merge until brave-core PR is approvable.** Link to brave-core PR:
  - [ ] Will you create brave-core PR to update to this commit after it is merged?
  - [ ] Wants uplift to brave-core feature branch?
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit

### CODE REVIEW

File: stories/features/rewards/other.tsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ storiesOf('Feature Components/Rewards/Other/Desktop', module)
           type={select<any>('Type', { big: 'big', small: 'small' }, 'big')}
           title={'Jonathon Doe'}
           verified={boolean('Verified', false)}
-          provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch' }, 'youtube')}
+          provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }, 'youtube')}
```
Comment (NejcZdovc) [inline stories/features/rewards/other.tsx]: Can you please change `{ youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }` to `{ youtube: 'youtube', twitter: 'twitter', twitch: 'twitch', reddit: 'reddit' }`

File: stories/features/rewards/wallet.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ storiesOf('Feature Components/Rewards/Wallet/Desktop', module)
       <div style={{ width: '373px', background: '#fff' }}>
         <WalletPanel
           id={'wallet-panel'}
-          platform={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch' }, 'youtube')}
+          platform={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }, 'youtube')}
```
Comment (NejcZdovc) [inline stories/features/rewards/wallet.tsx]: same as https://github.com/brave/brave-ui/pull/495/files#r291021780

File: stories/features/rewards/other.tsx [hunk 3]
```diff
@@ -233,7 +233,7 @@ storiesOf('Feature Components/Rewards/Other/Desktop', module)
         id={'toggle-tips'}
         onToggleTips={onToggle}
         tipsEnabled={boolean('Selected', store.state.tipsEnabled)}
-        provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch' }, 'youtube')}
+        provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }, 'youtube')}
```
Comment (NejcZdovc) [inline stories/features/rewards/other.tsx]: same as https://github.com/brave/brave-ui/pull/495/files#r291021780

File: stories/features/rewards/other.tsx (hunk 1 above)
Comment (jasonrsadler) [reply stories/features/rewards/other.tsx]: Done

File: stories/features/rewards/wallet.tsx (hunk 2 above)
Comment (jasonrsadler) [reply stories/features/rewards/wallet.tsx]: Done

File: stories/features/rewards/other.tsx (hunk 3 above)
Comment (jasonrsadler) [reply stories/features/rewards/other.tsx]: Done

**Pull request merged**